docs/pages/index.md

```diff
@@ -0,0 +1,51 @@
+# SKNNR Docs
+
+`scikit-learn-knn-regression` (a.k.a. `sknnr`) is a package for running k-nearest neighbor imputation methods, including GNN (Ohmann & Gregory, 2002), MSN (Moeur & Stage 1995), and RFNN (Crookston & Finley, 2008), using estimators that are fully compatible with [`scikit-learn`](https://scikit-learn.org/stable/).
```

Comment:
Having a really clear, one-sentence description of the package seems import here and in the readme. Any ideas to improve this?

Reply:
I will think about this, but I think your sentence (without the example estimators in it) seems really clear.  Perhaps we could follow on with a longer description that mentions the popular variants and history to provide some context.  I'm happy to try to draft something.

Reply:
Yeah, that would be great!

Reply:
OK, I took a first cut, but it may still be pretty rough.  I reorganized your first sentence under the heading "What is sknnr?", added a section called "Why the name 'sknnr'?" that explains the acronym, and added a section called "History and Inspiration" that details the tie to `yaImpute` and why we wanted to bring it into a python and scikit-learn context.

Reply:
The changes look great! `yaImpute` definitely deserved more of a mention than just a sentence in the acknowledgements, so adding the `History and Inspiration` section is a nice touch. The explanation of the name adds a lot of good context too.